src/lxc/utils.c

```diff
@@ -2040,14 +2050,14 @@ int lxc_safe_long(const char *numstr, long int *converted)
 
 	errno = 0;
 	sli = strtol(numstr, &err, 0);
-	if (errno > 0)
+	if (errno == ERANGE && (sli == LONG_MAX || sli == LONG_MIN))
 		return -errno;
 
-	if (!err || err == numstr || *err != '\0')
-		return -EINVAL;
+	if (errno != 0 && sli == 0)
```

Comment:
I see the manpage alluding to == 0 if there wer eno digits at all, but it only says that nptr original value will be returned in endptr, nothing about return value.  Do you have a pointer to better documentation about that?